app/views/course/assessment/assessments/show.html.slim

```diff
@@ -72,6 +72,10 @@
                       new_course_assessment_question_text_response_path(current_course, @assessment),
                       role: 'menuitem')
           li role='presentation'
+            = link_to(t('.new_question.voice_response'),
+              new_course_assessment_question_voice_response_path(current_course,
```

Comment:
Can follow the indentation style when we split method calls across lines?

Reply:
OK, will fix this.